recipes/godot_headers/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "3.2":
+    url: "https://github.com/godotengine/godot_headers/archive/ddf67cc7b8274c5fb77a71c828bab2991f1ee12a.zip"
```

Comment:
Please ask for tags to the upstream.

Reply:
https://github.com/godotengine/godot_headers/issues/70

Reply:
I think we shouldn't wait for upstream to tag their releases.
So what do we do?
Keep the top of the branch or switch to a commit?

Reply:
Using a commit is better for stability.

Reply:
If we go with a commit, then we have to go with the `cci.<date>` versioning.

Reply:
`cci.<date>` is fine with projects without releases because we reference commits in the main branch. `godot_headers` has official releases without tags, so we reference commits located in side branches. `cci.<date>` does not clearly designate the branch, nor the referenced version and may confuse consumers.

Reply:
So where does it clearly state which version is which commit? The branches can be updated, and probably will for patch releases

Reply:
I think we should just accept godot doesn't do releases and have to make sure that the commit hash is part of the correct branch ourselves.
I propose `cci.3.2.20200611` as version.

Reply:
`cci.3.2-20200611` is less confusing.